snippets/azure-oauth2-cambpm-web-4.5.0/README.md

```diff
@@ -0,0 +1,43 @@
+# azure-oauth2-cambpm-web-4.5.0
+
+Based on azure-oauth2-cambpm-web snippet
+
+## Introduction
+
+This project is a running example of a Camunda BPM Spring Boot instance using OAuth2 via Spring Security to authenticate a user against Azure Active Directory. 
+This project is using Spring Cloud Azure version +4.0.0 and a newer Camunda version.
+
+Main differences to older Azure AD example:
+
+- Spring Could Azure is integrated with Spring Security (no security client configuration necessary)
+- redirect-URI does not have /azure (security client name) suffix
+- antMatchers in WebAppSecurityConfig are set to "/camunda/app/**" which requires an authentication even for the welcome page
+- extra link configuration in config.js had to be adjusted, but is not necessary with the current setting of antMatchers
```

Comment:
if we merge the change, there is no need to mention the older example.

Also if the link configuration is not required I would remove it